packages/connect/src/http_client.ts

```diff
@@ -13,14 +13,20 @@ import {
     HttpRequestType,
     OrderbookRequest,
     OrderbookResponse,
-    OrdersRequest,
+    OrdersRequestOpts,
+    PagedRequestOpts,
     SignedOrder,
     TokenPairsItem,
-    TokenPairsRequest,
+    TokenPairsRequestOpts,
 } from './types';
 import { relayerResponseJsonParsers } from './utils/relayer_response_json_parsers';
 
 const TRAILING_SLASHES_REGEX = /\/+$/;
+const DEFAULT_PAGED_REQUEST_OPTS: PagedRequestOpts = {
+    page: 0,
+    per_page: 100,
```

Comment:
I'd prefer we use camelcase here `perPage` and then have a mapping `perPage: 'per_page'` which all the opts get looked up against before the query params get appended to the request URL.